_search-plugins/user-behavior-insights/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: User Behavior Insights
+parent: Search relevance
+has_children: false
+nav_order: 220
+---
+
+# User behavior insights
+
+User Behavior Insights, or UBI, is a community plugin for capturing client-side events and queries for the purposes of improving search relevance and user experience. UBI consists of three components:
+
+* An OpenSearch [plugin](https://github.com/o19s/opensearch-ubi) that facilitates the storage of client-side events and queries.
+* A client-side JavaScript library that demonstrates how to capture events and send those events to the OpenSearch UBI plugin.
```

Comment:
Could you elaborate on this more? I'm not sure "demonstrates" is the right verb here.